src/App.js

```diff
@@ -1,5 +1,22 @@
+import About from "./components/about/About";
+import Baking from "./components/baking/Baking";
+import Contact from "./components/contact/Contact";
+import Footer from "./components/footer/Footer";
+import Hero from "./components/hero/Hero";
+import Location from "./components/location/Location";
+import Navbar from "./components/navbar/Navbar";
+
 function App() {
-  return (<></>
+  return (
+  <>
+  <Navbar/>
+  <Hero/>
+  <About/>
+  <Location/>
+  <Baking/>
+  <Contact/>
+  <Footer/>
+  </>
```

Comment:
If you plan to have more pages in this app, then in the future you will need to move all this list of components to a separate one or it will mess the app.
If you don't plan to do this, then it's fine, but anyway, please add proper spaces here like

```
<>
    <Navbar/>
    <Hero/>
    <About/>
    <Location/>
    <Baking/>
    <Contact/>
    <Footer/>
<>
```